pkg/crypto/keys/publickey.go

```diff
@@ -242,6 +242,12 @@ func (p *PublicKey) EncodeBinary(w *io.BinWriter) {
 func (p *PublicKey) GetVerificationScript() []byte {
 	b := p.Bytes()
 	buf := io.NewBufBinWriter()
+	if address.Prefix == address.NEO2Prefix {
```

Comment:
Depending on `address` here is a bad idea.

Reply:
What do you suggest?

Reply:
I think we should just have all legacy decoding inside the cli package.